player.py

```diff
@@ -1,28 +1,56 @@
 import random
+import csv
+
 #from . import checkpointManager
 
 class Player:
     miles = random.randint(16,32)
     foodDaily = random.randint(12,15)
 
-    def __init__(self, character:str, money:int, food:int, huntAdjust:float, buyHudjust:float, distNext:int):
+    def __init__(self, character:str, money:int, food:int, huntAdjust:float, buyAudjust:float, distNext:int):
         self.character = character
         self.money = money
         self.food = food
         self.huntAdjust = huntAdjust
-        self.buyAdjust = buyHudjust
+        self.buyAdjust = buyAudjust
         self.distNext = distNext
 
-    def travel(self,didTravel=True, miles=miles, food=foodDaily):
-        #call scenario
-        if didTravel:
-            self.distNext -= miles
-            self.food -= food
-            if self.distNext <=0:
-                #call Checkpoint Manager
-                #checkpointManager
-                pass
+def selectCharacter():
+    filename = "characters.txt"
```

Comment:
If it is a csv it should have the CSV file type

Reply:
Updated